src/vcastAdapter.ts

```diff
@@ -88,55 +190,80 @@ export function deleteEnvironment(enviroPath: string, enviroNodeID: string) {
   );
 }
 
-// Load the Test Script into the Environment
-export async function loadScriptIntoEnvironment(
+// Load Test Script - server logic included -----------------------------------------
+export async function loadTestScriptIntoEnvironment(
   enviroName: string,
   scriptPath: string
 ) {
   // call clicast to load the test script
-  const enviroArg = `-e${enviroName}`;
-  let commandToRun: string = `${clicastCommandToUse} ${enviroArg} test script run ${scriptPath}`;
-
-  const commandStatus = executeCommandSync(
-    commandToRun,
-    path.dirname(scriptPath)
-  );
+  let loadScriptArgs: string = `-e${enviroName} test script run ${scriptPath}`;
+
+  let commandStatus: commandStatusType;
+  // using server ....
+  if (globalEnviroDataServerActive) {
+    const enviroPath = path.join(path.dirname(scriptPath), enviroName);
+    commandStatus = await executeClicastCommandUsingServer(
+      clicastCommandToUse,
```

Comment:
clicastCommandToUse argument is never used in the implementation of `executeClicastCommandUsingServer`

Reply:
Fixed with bdfc7e9